arrow-core/src/main/kotlin/arrow/core/extensions/nonemptylist/apply/NonEmptyListApply.kt

```diff
@@ -148,8 +125,8 @@ fun <A, B, Z> mapN(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "map(arg0, arg1, arg2, arg3)",
-  "arrow.core.NonEmptyList.map"
+    "NonEmptyList.tupledN(arg0, arg1, arg2).map(arg3)",
```

Comment:
These should be deprecated towards `NonEmptyList.mapN` and we should deprecate `tupeldN` towards `mapN`.

```suggestion
    "NonEmptyList.mapN(arg0, arg1, arg2) { a, b, c -> arg3(Tuple3(a, b, c)) }",
```